static/src/javascripts/es6/projects/common/modules/crosswords/grid.js

```diff
@@ -6,67 +6,16 @@ import React from 'react';
 
 import _ from 'common/utils/_';
 
-import constants from 'es6/projects/common/modules/crosswords/constants';
-import helpers from 'es6/projects/common/modules/crosswords/helpers';
+import constants from './constants';
+import helpers from './helpers';
 
-const Cell = React.createClass({
-    onClick (event) {
-        event.preventDefault();
-        this.props.handleSelect();
-    },
+import Cell from './cell';
 
-    render () {
-        const innerNodes = [];
-        const top = helpers.gridSize(this.props.y);
-        const left = helpers.gridSize(this.props.x);
-
-        if (this.props.number !== undefined) {
-            innerNodes.push(
-                React.DOM.text({
-                    x: left + 1,
-                    y: top + constants.numberSize,
-                    key: 'number',
-                    className: 'crossword__cell-number'
-                }, this.props.number)
-            );
-        }
-
-        if (this.props.value !== undefined) {
-            innerNodes.push(React.DOM.text({
-                x: left + constants.cellSize / 2,
-                y: top + (constants.cellSize + constants.entrySize) / 2,
-                key: 'entry',
-                className: classNames({
-                    'crossword__cell-text': true,
-                    'crossword__cell-text--error': this.props.isError
-                })
-            }, this.props.value));
-        }
-
-        return React.DOM.g({
-            onClick: this.onClick
-        },
-            React.DOM.rect({
-                x: left,
-                y: top,
-                width: constants.cellSize,
-                height: constants.cellSize,
-                className: classNames({
-                    'crossword__cell': true,
-                    'crossword__cell--focussed': this.props.isFocussed,
-                    'crossword__cell--highlighted': this.props.isHighlighted,
-                    'crossword__cell--intersecting': this.props.intersectsFocussedEntry
-                })
-            }),
-            innerNodes
-        );
-    }
-});
+export default class extends React.Component {
 
-export default React.createClass({
     handleSelect (x, y) {
```

Comment:
I know this was me, but not really any need for this given it just delegates immediately to the props function